ui/packages/tidb-dashboard-lib/src/apps/Statement/utils/useStatementTableController.ts

```diff
@@ -235,6 +248,11 @@ export default function useStatementTableController({
       setDataLoading(true)
 
       const timeRange = toTimeRangeValue(queryOptions.timeRange)
+      // enlarge the time range automatically for old tidb-dashboard backend
+      if (oldBackend) {
+        timeRange[0] -= windowSize
+        timeRange[1] += windowSize
+      }
```

Comment:
Is that ok for the latest api at first? If it is good, why not just use the expanded time range?

Reply:
> Is that ok for the latest api at first?

for the new api, after we check it is the new api by the first request, it will change `oldBackend` from `true` to `false`, and trigger re-render() and re-request again. so the list result can be fixed very quickly.

> why not just use the exapended time range?

I think use the original time range for the new api can make the result more accurate.